src/index.js

```diff
@@ -4,8 +4,8 @@ import cx from 'classnames';
 import debounce from './lib/debounce';
 
 class OnVisible extends Component {
-    constructor() {
-        super(...arguments);
+    constructor(props) {
+        super(props);
```

Comment:
think should leave the super invocation with all arguments in - possibly subtle bugs might crawl in if not.